### PR TITLE
fix: (Core) cursor on disabled Switch should not be pointer

### PR DESCRIPTION
**Pull request by olshansky**
#### Please provide a link to the associated issue.
fixes https://github.com/SAP/fundamental-ngx/issues/3720
#### Please provide a brief summary of this pull request.
switch-disabled

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/core/src/lib/switch/switch.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<label class="fd-switch__label" (click)="focus()">
+<label class="fd-switch__label" (click)="focus()" [ngStyle]="{ cursor: disabled ? 'not-allowed' : 'pointer'}">
```
Comment (JKMarkowski) [inline libs/core/src/lib/switch/switch.component.html:1]: I think this cursor should be auto

**Review by salarenko — CHANGES_REQUESTED** [review]
We should just patch the styles here:
```
.fd-switch.is-disabled {
    cursor: not-allowed;
}
```

**Comment by olshansky** [comment]
> We should just patch the styles here:
> 
> ```
> .fd-switch.is-disabled {
>     cursor: not-allowed;
> }
> ```

@salarenko unfortunately, it won't work
switch-disabled

File: libs/core/src/lib/switch/switch.component.html (hunk 1 above)
Comment (olshansky) [reply libs/core/src/lib/switch/switch.component.html]: @JKMarkowski fixed

**Review by salarenko — APPROVED** [review]

**Review by droshev — APPROVED** [review]

**Pull request merged**